2021-08-26 07:09:52,298 INFO    MainThread:23954 [wandb_setup.py:_flush():69] setting env: {}
2021-08-26 07:09:52,298 INFO    MainThread:23954 [wandb_setup.py:_flush():69] setting login settings: {}
2021-08-26 07:09:52,299 INFO    MainThread:23954 [wandb_init.py:_log_setup():342] Logging user logs to /opt/ml/image-classification-level1-25/wandb/run-20210826_070952-rsjbb2c3/logs/debug.log
2021-08-26 07:09:52,299 INFO    MainThread:23954 [wandb_init.py:_log_setup():343] Logging internal logs to /opt/ml/image-classification-level1-25/wandb/run-20210826_070952-rsjbb2c3/logs/debug-internal.log
2021-08-26 07:09:52,300 INFO    MainThread:23954 [wandb_init.py:init():375] calling init triggers
2021-08-26 07:09:52,300 INFO    MainThread:23954 [wandb_init.py:init():380] wandb.init called with sweep_config: {}
config: {'epochs': 20, 'batch_size': 32, 'learning_rate': 0.001}
2021-08-26 07:09:52,300 INFO    MainThread:23954 [wandb_init.py:init():395] re-initializing run, found existing run on stack: 3pmwikv0
2021-08-26 07:09:52,302 INFO    MainThread:23954 [wandb_run.py:finish():1254] finishing run presto105/lv1_p/3pmwikv0
2021-08-26 07:09:52,341 INFO    MainThread:23954 [jupyter.py:save_history():339] not saving jupyter history
2021-08-26 07:09:52,341 INFO    MainThread:23954 [jupyter.py:save_ipynb():284] not saving jupyter notebook
2021-08-26 07:09:52,341 INFO    MainThread:23954 [wandb_init.py:_jupyter_teardown():275] cleaning up jupyter logic
2021-08-26 07:09:52,342 INFO    MainThread:23954 [wandb_run.py:_atexit_cleanup():1554] got exitcode: 0
2021-08-26 07:09:52,342 INFO    MainThread:23954 [wandb_run.py:_restore():1526] restore
2021-08-26 07:09:53,020 INFO    MainThread:23954 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1081
  total_bytes: 1081
}

2021-08-26 07:09:53,312 INFO    MainThread:23954 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1081
  total_bytes: 1081
}

2021-08-26 07:09:53,536 INFO    MainThread:23954 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 3
}
pusher_stats {
  uploaded_bytes: 1081
  total_bytes: 9429
}

2021-08-26 07:09:53,639 INFO    MainThread:23954 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1081
  total_bytes: 9778
}

2021-08-26 07:09:53,741 INFO    MainThread:23954 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1081
  total_bytes: 9778
}

2021-08-26 07:09:53,843 INFO    MainThread:23954 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6584
  total_bytes: 9778
}

2021-08-26 07:09:53,946 INFO    MainThread:23954 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 9778
  total_bytes: 9778
}

2021-08-26 07:09:54,049 INFO    MainThread:23954 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 9778
  total_bytes: 9778
}

2021-08-26 07:09:54,151 INFO    MainThread:23954 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 9778
  total_bytes: 9778
}

2021-08-26 07:09:54,253 INFO    MainThread:23954 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 9778
  total_bytes: 9778
}

2021-08-26 07:09:54,357 INFO    MainThread:23954 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 9778
  total_bytes: 9778
}

2021-08-26 07:09:54,752 INFO    MainThread:23954 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 9778
  total_bytes: 9778
}

2021-08-26 07:09:54,854 INFO    MainThread:23954 [wandb_run.py:_wait_for_finish():1676] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 9778
  total_bytes: 9778
}

2021-08-26 07:09:55,992 INFO    MainThread:23954 [wandb_run.py:_show_files():1898] logging synced files
2021-08-26 07:09:55,996 INFO    MainThread:23954 [wandb_init.py:init():424] starting backend
2021-08-26 07:09:55,997 INFO    MainThread:23954 [backend.py:_multiprocessing_setup():70] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-08-26 07:09:56,000 INFO    MainThread:23954 [backend.py:ensure_launched():135] starting backend process...
2021-08-26 07:09:56,011 INFO    MainThread:23954 [backend.py:ensure_launched():139] started backend process with pid: 24138
2021-08-26 07:09:56,012 INFO    MainThread:23954 [wandb_init.py:init():429] backend started and connected
2021-08-26 07:09:56,028 INFO    MainThread:23954 [wandb_run.py:_label_probe_notebook():798] probe notebook
2021-08-26 07:09:56,029 INFO    MainThread:23954 [wandb_run.py:_label_probe_notebook():805] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2021-08-26 07:09:56,029 INFO    MainThread:23954 [wandb_init.py:init():477] updated telemetry
2021-08-26 07:09:56,030 INFO    MainThread:23954 [wandb_init.py:init():500] communicating current version
2021-08-26 07:09:56,798 INFO    MainThread:23954 [wandb_init.py:init():505] got version response 
2021-08-26 07:09:56,798 INFO    MainThread:23954 [wandb_init.py:init():513] communicating run to backend with 30 second timeout
2021-08-26 07:09:57,016 INFO    MainThread:23954 [wandb_init.py:init():540] starting run threads in backend
2021-08-26 07:10:01,418 INFO    MainThread:23954 [wandb_run.py:_console_start():1584] atexit reg
2021-08-26 07:10:01,418 INFO    MainThread:23954 [wandb_run.py:_redirect():1458] redirect: SettingsConsole.WRAP
2021-08-26 07:10:01,419 INFO    MainThread:23954 [wandb_run.py:_redirect():1495] Wrapping output streams.
2021-08-26 07:10:01,420 INFO    MainThread:23954 [wandb_run.py:_redirect():1519] Redirects installed.
2021-08-26 07:10:01,420 INFO    MainThread:23954 [wandb_init.py:init():565] run started, returning control to user process
2021-08-26 07:10:12,521 INFO    MainThread:23954 [wandb_setup.py:_flush():69] setting env: {}
2021-08-26 07:10:12,522 INFO    MainThread:23954 [wandb_setup.py:_flush():69] setting login settings: {}
2021-08-26 07:10:12,523 INFO    MainThread:23954 [wandb_init.py:_log_setup():342] Logging user logs to /opt/ml/image-classification-level1-25/wandb/run-20210826_071012-17xbgkfe/logs/debug.log
2021-08-26 07:10:12,523 INFO    MainThread:23954 [wandb_init.py:_log_setup():343] Logging internal logs to /opt/ml/image-classification-level1-25/wandb/run-20210826_071012-17xbgkfe/logs/debug-internal.log
2021-08-26 07:10:12,524 INFO    MainThread:23954 [wandb_init.py:_jupyter_setup():294] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7ff6816d5c10>
2021-08-26 07:10:12,525 INFO    MainThread:23954 [wandb_init.py:init():375] calling init triggers
2021-08-26 07:10:12,525 INFO    MainThread:23954 [wandb_init.py:init():380] wandb.init called with sweep_config: {}
config: {'epochs': 20, 'batch_size': 32, 'learning_rate': 0.001}
2021-08-26 07:10:12,525 INFO    MainThread:23954 [wandb_init.py:init():395] re-initializing run, found existing run on stack: rsjbb2c3
2021-08-26 07:10:12,527 INFO    MainThread:23954 [wandb_run.py:finish():1254] finishing run presto105/lv1_p/rsjbb2c3
